dspace-api/src/main/java/org/dspace/statistics/util/SpiderDetectorServiceImpl.java

```diff
@@ -185,24 +185,27 @@ private void loadPatterns(String directory, List<Pattern> patternList) {
         File patternsDir = new File(spidersDir, directory);
         if (patternsDir.exists() && patternsDir.isDirectory()) {
             for (File file : patternsDir.listFiles()) {
-                Set<String> patterns;
-                try {
-                    patterns = readPatterns(file);
-                } catch (IOException ex) {
-                    log.error("Patterns not read from {}:  {}",
-                              file.getPath(), ex.getMessage());
-                    continue;
-                }
-                //If case insensitive matching is enabled, lowercase the patterns so they can be lowercase matched
-                for (String pattern : patterns) {
-                    if (isUseCaseInsensitiveMatching()) {
-                        pattern = StringUtils.lowerCase(pattern);
+                if (file.isFile()
+                    && !file.getName().matches(".+old$")) { // UH omit .old files from previous update runs
```

Comment:
Just a note, we'd need to remove these comments that say "UH".  The comment itself is perfectly OK, as long as you remove the "UH". 

There's a second comment in this same file that also has "UH" at the beginning.